src/accelerate/tracking.py

```diff
@@ -681,12 +685,187 @@ def finish(self):
         mlflow.end_run()
 
 
+class ClearMLTracker(GeneralTracker):
+    """
+    A `Tracker` class that supports `clearml`. Should be initialized at the start of your script.
+
+    Environment:
+    - **CLEARML_PROJECT** (`str`, *optional*) - The default ClearML project name. Can be overwritten by setting
+      `project_name` in `task_init_kwargs`.
+    - **CLEARML_TASK** (`str`, *optional*) - The default ClearML task name. Can be overwritten by setting `task_name`
+      in `task_init_kwargs`.
+
+    Args:
+        run_name (`str`, *optional*):
+            Name of the experiment. If ClearML's `Task.init`'s `task_name` and `project_name` are not specified in
+            kwargs, they will default to this value
+        task_init_kwargs:
+            Kwargs passed along to the `Run.__init__` method.
+    """
+
+    name = "clearml"
+    requires_logging_directory = False
+
+    @on_main_process
+    def __init__(self, run_name: str = None, **task_init_kwargs):
+        from clearml import Task
+
+        current_task = Task.current_task()
+        if current_task:
+            self._initialized_externally = True
+            self.task = current_task
+            return
+
+        if "CLEARML_PROJECT" in os.environ:
+            task_init_kwargs.setdefault("project_name", os.environ["CLEARML_PROJECT"])
+        else:
+            task_init_kwargs.setdefault("project_name", run_name)
+        if "CLEARML_TASK" in os.environ:
+            task_init_kwargs.setdefault("task_name", os.environ["CLEARML_TASK"])
+        else:
+            task_init_kwargs.setdefault("task_name", run_name)
+        self.task = Task.init(**task_init_kwargs)
+
+    @property
+    def tracker(self):
+        return self.task
+
+    @on_main_process
+    def store_init_configuration(self, values: dict):
+        """
+        Connect configuration dictionary to the Task object. Should be run at the beginning of your experiment.
+
+        Args:
+            values (`dict`):
+                Values to be stored as initial hyperparameters as key-value pairs.
+        """
+        if not self.task:
```

Comment:
Can you explain why this would ever be called?

Reply:
Paranoia got the best of me :). Indeed, this should never be called. I removed this section